website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,103 @@
+$(document).ready(function () {
+	console.log(openerp);
+	$('.type_selection').each(function () {
+	    var type_selection = this;
+	    $(type_selection).on('click', '.type_selection', function ()
+	    {
+	    	openerp.jsonRpc('/chd_init/getch/', 'call',
+	    		 {
+	             'type_id': this.value,
+	    		  }).then(function (data) {
```

Comment:
stick to one way of formatting inline functions. I like the way you do it in line 6ff